src/main/java/com/demo/classroom/Service/AuthService.java

```diff
@@ -41,21 +41,24 @@ public class AuthService {
 
     private AuthenticationManager authenticationManager;
 
-    @Autowired
     private JwtService jwtService;
-    @Autowired
+
     private UserService userService;
 
     @Autowired
-    public AuthService(UserRepository userRepository, TeacherRepository teacherRepository, 
-        StudentRepository studentRepository, PasswordEncoder passwordEncoder,
+    public AuthService(
+        UserRepository userRepository, UserService userService, JwtService jwtService,
+        TeacherRepository teacherRepository, StudentRepository studentRepository, 
+        PasswordEncoder passwordEncoder,
```

Comment:
Can we find a cleaner approach to add parameters?

Reply:
done